package.json

```diff
@@ -108,7 +108,18 @@
     ],
     "mac": {
       "category": "public.app-category.productivity",
-      "minimumSystemVersion": "10.12.0"
+      "minimumSystemVersion": "10.12.0",
+      "extendInfo": {
+        "CFBundleDocumentTypes": [
+          {
+            "CFBundleTypeExtensions": ["mp4"],
+            "CFBundleTypeName": "MP4",
+            "CFBundleTypeRole": "Editor",
+            "CFBundleTypeIconFile": "Kap.icns",
+            "LSHandlerRank": "Alternate"
```

Comment:
Should we support MOV too?

`CFBundleTypeExtensions` is deprecated.

See: https://github.com/sindresorhus/gifski-app/blob/28948889a8a544c731db9cd29cec98423882da3f/Gifski/Info.plist#L10-L21

Reply:
Hmm, good to know @sindresorhus. Electron-builder still uses it. That's the reason I went for it. I'll update it